Equipment/EquipmentDb/Misc/rings.xml

```diff
@@ -437,6 +437,18 @@
 		</source>
 	</item>
 
+	<item id="133366" phase="3">
```

Comment:
This should be 19434.

Reply:
Aha, I think the id I used is the icon id.